ca_description/urdf/create_base.xacro

```diff
@@ -44,6 +45,24 @@
       </collision>
     </link>
 
+    <xacro:property name="diameter" value="0.3485"/>
+    <xacro:property name="size_z" value="0.085"/>
+    
+    <!-- Creating the constants to introduce origin -->
+
+    <xacro:property name="left_origin">
+      <origin xyz="${diameter/2 * 0.5 + 0.01} ${diameter/2 * 0.87 + 0.01} ${size_z}" rpy="0 ${pi/4} 0" />
```

Comment:
This is not the correct way of passing Xacro block parameters. Again, [here](http://wiki.ros.org/xacro#Macros) is the link.